src/site/content/en/blog/coop-coep/index.md

```diff
@@ -23,27 +23,40 @@ origin_trial:
 feedback:
   - api
 ---
+{% Aside 'gotchas' %}
+
+**[15th Oct. 2020]**: `self.crossOriginIsolated` is available from Chrome 87.
+Reflecting that, `document.domain` is immutable when `self.crossOriginIsolated`
+returns `true`. `performance.measureMemory` ending its origin trial and is
+planned to be enabled by default in Chrome 88. Shared Array Buffer on Android
+Chrome will be available from Chrome 88.
+
+**[1st Sep. 2020]**: COOP Reporting is behind the `#cross-origin-isolated` flag
+in Chrome 86. Added instructions.
+
+{% endAside %}
```

Comment:
```suggestion
{% Banner 'caution', 'body' %}

## Updates

### October 15th, 2020

`self.crossOriginIsolated` is available from Chrome 87.
Reflecting that, `document.domain` is immutable when `self.crossOriginIsolated`
returns `true`. `performance.measureMemory` ending its origin trial and is
planned to be enabled by default in Chrome 88. Shared Array Buffer on Android
Chrome will be available from Chrome 88.

### September 1st, 2020

COOP Reporting is behind the `#cross-origin-isolated` flag
in Chrome 86.

{% endBanner %}
```

I'm not seeing any instructions for the `#cross-origin-isolated` on this page. Also, please avoid `Added instructions` and use our common phrasing: `See [Section name](#link)`. 

Let's try using a body banner here rather than a gotcha: https://web.dev/handbook/web-dev-components/#body-banners

LMK if you want me to explain why a gotcha doesn't seem appropriate here